### PR TITLE
Image adjustment nodes restructure

### PR DESCRIPTION
**Pull request by TrueDoctor**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (Keavon) [inline package-lock.json:1]: Revert

File: editor/src/messages/portfolio/document/node_graph/node_graph_message_handler/node_properties.rs [hunk 2]
```diff
@@ -189,6 +189,18 @@ pub fn adjust_gamma_properties(document_node: &DocumentNode, node_id: NodeId, _c
 	vec![LayoutGroup::Row { widgets: gamma }]
 }
 
+pub fn adjust_threshold_properties(document_node: &DocumentNode, node_id: NodeId, _context: &mut NodePropertiesContext) -> Vec<LayoutGroup> {
+	let thereshold = number_widget(document_node, node_id, 1, "Threshold", NumberInput::default().min(0.00).max(1.), true);
```
Comment (Keavon) [inline editor/src/messages/portfolio/document/node_graph/node_graph_message_handler/node_properties.rs:193]: .clamp(0., 1.)

File: editor/src/messages/portfolio/document/node_graph/node_graph_message_handler/node_properties.rs [hunk 3]
```diff
@@ -189,6 +189,18 @@ pub fn adjust_gamma_properties(document_node: &DocumentNode, node_id: NodeId, _c
...
+
+	vec![LayoutGroup::Row { widgets: thereshold }]
+}
+
+pub fn adjust_vibrance_properties(document_node: &DocumentNode, node_id: NodeId, _context: &mut NodePropertiesContext) -> Vec<LayoutGroup> {
+	let vibrance = number_widget(document_node, node_id, 1, "Vibrance", NumberInput::default().min(-100.).max(100.).unit("%"), true);
```
Comment (Keavon) [inline editor/src/messages/portfolio/document/node_graph/node_graph_message_handler/node_properties.rs:199]: .clamp(-100., 100.)

File: editor/src/messages/portfolio/document/node_graph/node_graph_message_handler/node_properties.rs (hunk 2 above)
Comment (TrueDoctor) [reply editor/src/messages/portfolio/document/node_graph/node_graph_message_handler/node_properties.rs:193]: Not sure what you mean, `.clamp()` does not exist for NumberInputs afaik

Comment (Keavon) [reply editor/src/messages/portfolio/document/node_graph/node_graph_message_handler/node_properties.rs:193]: Oh, derp. I thought this was f32 min and max functions.

**Review by Keavon — APPROVED** [review]

**Pull request merged**